.github/workflows/test_onnxruntime_gpu.yml

```diff
@@ -2,11 +2,14 @@ name: ONNX Runtime / Test GPU
 
 on:
   workflow_dispatch:
-  push:
-    branches:
-      - main
   schedule:
     - cron: 0 7 * * * # every day at 7am
+  pull_request:
+    types: [labeled]
```

Comment:
It can be any label right?

Reply:
Yes, I added it so that the test can be triggered on a PR. I don't know if we can specify a specific tag, do you know?